theme/views.py

```diff
@@ -120,6 +122,8 @@ def rating(request):
     obj, post_data = response
     url = add_cache_bypass(obj.get_absolute_url().split("#")[0])
     response = redirect(url + "#rating-%s" % obj.id)
+    resource_mode = post_data.get('resource-mode', 'view')
+    request.session['resource-mode'] = resource_mode
```

Comment:
Yes, I saw that and commented the specific line where the ```resource-mode``` session variable is deleted, although I am a bit confused on the logic there specifically on why edit-resource variable needs to be assigned twice.